task_router_activity.go

```diff
@@ -0,0 +1,80 @@
+package twilio
+
+import (
+	"context"
+	"net/url"
+)
+
+const activityPathPart = "Activities"
+
+type TaskRouterActivityService struct {
+	client *Client
+}
+
+type Activity struct {
+	Sid          string     `json:"sid"`
+	AccountSid   string     `json:"account_sid"`
+	FriendlyName string     `json:"friendly_name"`
+	Available    bool       `json:"available"`
+	DateCreated  TwilioTime `json:"date_created"`
+	DateUpdated  TwilioTime `json:"date_updated"`
+	URL          string     `json:"url"`
+	WorkspaceSid string     `json:"workspace_sid"`
+}
+
+type ActivityPage struct {
+	Page
+	Activities []*Activity `json:"activities"`
+}
+
+func (r *TaskRouterActivityService) Get(ctx context.Context, sid string) (*Activity, error) {
+	activity := new(Activity)
+	err := r.client.GetResource(ctx, activityPathPart, sid, activity)
+	return activity, err
+}
+
+func (r *TaskRouterActivityService) Create(ctx context.Context, data url.Values) (*Activity, error) {
```

Comment:
As an example, I'm consuming this API:

```
data := url.Values{}
data.Set("FriendlyName", d.Get("name").(string))

activity, err := m.Client.TaskRouter.Activities.Create(context.Background(), data)
```

Because `data` is type `url.Values` what is the recommend way of updating the boolean `Available` field? As `Values.set` only accepts a string - https://golang.org/pkg/net/url/#Values.Set

Reply:
Diving further into this, I've had a look at other Go API packages and they seem to define a `SomeNameCreateRequest` struct and turning this into JSON. See an example below:

https://github.com/digitalocean/godo/blob/master/droplets.go#L242
https://github.com/digitalocean/godo/blob/master/droplets.go#L344

I've had a look around this package at it seems we have other places where we define a non `string` type and try and pass it in via `url.Values`. For example `applications.go` takes a `VoiceCallerIDLookup bool`, so what's the recommended way of updating this field when using  `url.Values{}`

Reply:
Should work to just do `Set("Available", "true")`. You can always have "true" as a string

Reply:
We're turning this back into form-urlencoded strings to send over the wire so I'd rather leave it that way :(